core/util/src/main/java/edu/cmu/cs/dennisc/java/awt/RectangleUtilities.java

```diff
@@ -54,21 +51,6 @@
  * @author Dennis Cosgrove
  */
 public class RectangleUtilities {
```

Comment:
i wouldn't be mad if this entire file could go away, but you're doing good work here

Reply:
The remaining methods are mostly used in only one other file, so they could be moved to those those files.
The inset method is the one exception. It is used in a few places, but looks a lot like `Rectangle.grow()`, so maybe it could all go in a future round?